tests/Tests/700__sandbox/701__perfromance_sandbox.robot

```diff
@@ -0,0 +1,81 @@
+*** Settings ***
+Documentation    Test Suite for Performance Sandbox Test cases
+Library         OperatingSystem
+Library         Collections
+Library         Process
+Library         String
+
+
+*** Variables ***
+${NAMESPACE}     openshift-kube-apiserver
+${LABEL_SELECTOR}     app=openshift-kube-apiserver
+${MEMORY_THRESHOLD}    102400
+${PERF_CODE}    go run setup/main.go --users 2000 --default 2000  --custom 0 --username "user" --workloads redhat-ods-operator:rhods-operator --workloads redhat-ods-applications:rhods-dashboard --workloads redhat-ods-operator:cloud-resource-operator --workloads redhat-ods-monitoring:blackbox-exporter --workloads redhat-ods-monitoring:grafana --workloads redhat-ods-monitoring:prometheus <<< y   #robocop:disable
+
+
+*** Test Cases ***
+Verify RHODS Performance For Sandbox Onboarding Process
+    [Documentation]    Performance test for RHODS operator onboard
+    ...    to Sandbox environment
+    [Tags]     ODS-1404
+    ...        Sandbox
+    ...        Performance-Test
+    Run    git clone https://github.com/codeready-toolchain/toolchain-e2e.git
+    Run Keyword And Continue On Failure    Run Performance Test On RHODS Operator
+    Verify Sandbox Toolchain Data
+
+
+*** Keywords ***
+Capture Memory Utilization Of Openshift API Server POD
+    [Documentation]  Capture and compare the realtime memory utilization
+    ${memory_usage}     Run    kubectl top pod -n ${NAMESPACE} -l ${LABEL_SELECTOR} | awk '{if(NR>1)print $3}'
```

Comment:
would it be the same using `oc` instead of kubectl? just asking since we will replace all the calls to `oc` with openshiftlibrary